etc/nginx/http.d/01-deskpro_setup.conf.tmpl

```diff
@@ -1,9 +1,9 @@
 upstream dpv5_api {
-  server unix:/run/php_fpm_dp_default.sock;
+  server unix:/run/php_fpm_dp_default.sock max_fails=0 fail_timeout=0;
```

Comment:
We want to make sure an error on this upstream never "takes it out of rotation". It's local, so it should always be tried.